django/roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Install nginx
+  ansible.builtin.apt:
+    pkg:
+     - nginx
+    state: present
+    update_cache: true
+
+- name: Configure Nginx
+  ansible.builtin.template:
+    src: nginx.j2
+    dest: /etc/nginx/sites-enabled/default
```

Comment:
make sure to create custom configuration files.
do not use default.

Reply:
Done.